extensions/wikia/Email/Controller/UserNameChangeController.class.php

```diff
@@ -42,21 +42,33 @@ public function body() {
 			'editorAvatarURL' => $this->getCurrentAvatarURL(),
 			'summary' => $this->getMessage( 'emailext-usernamechange-summary' )->text(),
 			'userNameMessage' => $this->getMessage( 'emailext-usernamechange-changed',
-				$this->oldUserName, $this->newUserName )->text(),
+				$this->oldUserName, $this->newUserName )->parse(),
+			'links' => $this->generateLinks(),
 			'contentFooterMessages' => [
 				$this->getMessage( 'emailext-usernamechange-closing' )->text(),
-				$this->getMessage( 'emailext-usernamechange-signature' )->text()
 			],
-			'hasContentFooterMessages' => true
+			'signature' => $this->getMessage( 'emailext-usernamechange-signature' )->text()
 		] );
 	}
 
 	protected function getSubject() {
 		return $this->getMessage( 'emailext-usernamechange-subject' )->text();
 	}
 
+	private function generateLinks() {
+		return [
+			[
+				'label' => $this->getMessage( 'emailext-usernamechange-profile-page' )->text(),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "label" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)